### PR TITLE
Improved documentation + small fix

### PR DESCRIPTION
**Pull request by tycrek**
Mostly improvements to the README (better formatting, more links/info).

Some lines in `pom.xml` switched from referencing `Steveice10/MCAuthLib` to `GeyserMC/MCAuthLib` because the old repository redirects here anyways, so the Maven info may as well have this updated information too.

Last thing: small fix in `HTTP.java`, I had to remove an unused `import` that was causing compilation to fail.

### CODE REVIEW

**Comment by rtm516** [comment]
The jitpack links can be replaced with the GeyserMC org as it seems to have a few builds done already
https://jitpack.io/#GeyserMC/MCAuthLib

**Comment by tycrek** [comment]
Updated 👍

File: pom.xml [hunk 1]
```diff
@@ -10,18 +10,18 @@
...
+        <connection>scm:git:git@github.com:GeyserMC/MCAuthLib.git</connection>
+        <developerConnection>scm:git:git@github.com:GeyserMC/MCAuthLib.git</developerConnection>
+        <url>git@github.com:GeyserMC/MCAuthLib/</url>
     </scm>
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <jdk.version>1.7</jdk.version>
-        <argLine></argLine>
+        <argLine/>
```
Comment (rtm516) [inline pom.xml]: Gonna presume this wasn't meant to change, revert for a cleaner commit and makes it easier for people that need to change it later.
```suggestion
        <argLine></argLine>
```

**Review by rtm516 — CHANGES_REQUESTED** [review]

**Comment by tycrek** [comment]
Was fixed in 76fa778, not sure if GitHub sent any alerts for it (tried accepting changes from the app for the first time)

**Comment by tycrek** [comment]
@rtm516 Able to re-review this one?

**Review by rtm516 — APPROVED** [review]

**Pull request closed without merging**